### PR TITLE
docs: documented the Wayland gateway

### PR DESCRIPTION
**Pull request by e8johan**
Just added docs for the Wayland gateway.

### CODE REVIEW

File: doc/chapters/gateways/index.rst [hunk 1]
```diff
@@ -422,7 +422,16 @@ Wayland gateway
 
 The Wayland Gateway is used to provide access to the hsot system Wayland server.
 
+Please notice that the Wayland Gateway only provides access to the Wayland
+socket and sets the ``XDG_RUNTIME_DIR`` accordinly. For a Wayland
```
Comment (frznlogic) [inline doc/chapters/gateways/index.rst]: Accordingly.

**Review by frznlogic — CHANGES_REQUESTED** [review]
No. ;)

File: doc/chapters/gateways/index.rst (hunk 1 above)
Comment (tobsan) [inline doc/chapters/gateways/index.rst]: accordingly*

File: doc/chapters/gateways/index.rst [hunk 2]
```diff
@@ -422,7 +422,16 @@ Wayland gateway
 
 The Wayland Gateway is used to provide access to the hsot system Wayland server.
 
+Please notice that the Wayland Gateway only provides access to the Wayland
```
Comment (frznlogic) [inline doc/chapters/gateways/index.rst]: s/notice/note/ ... ?

File: doc/chapters/gateways/index.rst [hunk 3]
```diff
@@ -422,7 +422,16 @@ Wayland gateway
...
+socket and sets the ``XDG_RUNTIME_DIR`` accordinly. For a Wayland
+*capability*, it is also necessary to ensure that the contained application
+has access to the graphics hardware devices and associated libraries.
```
Comment (tobsan) [inline doc/chapters/gateways/index.rst:428]: Actually, access to hardware devices does not seem to be necessary, at least not when testing. We are using the wayland gateway and running "weston-info" inside the container, which works. No device nodes are copied into the container. Perhaps the test is too rudimentary, I don't know.

**Review by tobsan — CHANGES_REQUESTED** [review]

File: doc/chapters/gateways/index.rst (hunk 3 above)
Comment (e8johan) [reply doc/chapters/gateways/index.rst:428]: The test only tries talking to the server, it does not try to do graphics. The test is too rudimentary for that.

File: doc/chapters/gateways/index.rst (hunk 1 above)
Comment (e8johan) [reply doc/chapters/gateways/index.rst]: fixing

File: doc/chapters/gateways/index.rst (hunk 2 above)
Comment (e8johan) [reply doc/chapters/gateways/index.rst]: fixing

File: doc/chapters/gateways/index.rst (hunk 1 above)
Comment (e8johan) [reply doc/chapters/gateways/index.rst]: fixing

**Review by frznlogic — APPROVED** [review]

**Pull request merged**